### DISCUSSION TITLE
「鬥」のコメントについて、および変換候補に部首を表示させる際のコメントについて

### DISCUSSION THREAD

**Discussion by ghost** [google/mozc#1471] [Ideas]
「たたかいがまえ」で変換すると、以下の候補が表示されます。
```
＜前略＞
鬥 [たたかいがまえ とうがまえ]
```
この候補に表示される鬥は、部首のたたかいがまえ (U+2FBE) ではなく、漢字の「たたかう」(U+92B5)です。この文字が部首であるかと誤認させるコメントは改める必要があるのではないでしょうか。また、部首を候補に表示させるときは、それが（Unicode的に）漢字ではないことを明示するために、コメントに「部首」という文言を挿入するのが望ましいと考えています。

**Comment by ghost** [comment]
同様の問題は「くさかんむり」の変換候補にもあります。くさかんむりで変換すると、候補に「艸(U+8278)」が現れます。しかしこれは部首の草冠 (U+2F8B)ではなく、漢字のくさです。

**Comment by ghost** [comment]
全般的に、Mozcの変換候補はUnicodeの漢字と部首を混同しているように見受けられます。抜本的に改善する必要があると考えます。

**Comment by tats-u** [comment]
康煕部首・CJK部首補助ブロックは日本の規格が元になったものではないはずです。
多分中国の要望のはず
なので日本のIMEが通常の漢字ブロックに入っている部首漢字を出すのは合理的だと思います。
